src/app/routes/activation.ts

```diff
@@ -16,16 +16,28 @@ class ActivationController {
     this.service = service;
   }
 
+  async sendDeactivationEmail(req: Request<{ email: string }>, res: Response) {
+    const email = req.params.email.toLowerCase();
+    const user = await this.service.User.FindUserByEmail(email);
+    const uuid = user.uuid;
+
+    await this.service.User.deactivate(email);
+
+    AnalyticsService.trackDeactivationRequest(uuid, req);
```

Comment:
Try to put the analytics always after responding to the client, to avoid interfering with the expected server behavior